codebase/superdarn/src.bin/tk/plot/field_plot.1.12/doc/field_plot.doc.xml

```diff
@@ -92,6 +92,10 @@
 </option>
 <option><on>-flip</on><od>flip the direction of the X-axis.</od>
 </option>
+<option><on>-xshift</on><od>shift the plot horizontally by a percentage of the plotting region (+/- 1.0)</od>
+</option>
+<option><on>-yshift</on><od>shift the plot vertically by a percentage of the plotting region (+/- 1.0)</od>
+</option>
```

Comment:
Would this be better wording since the shift values are in the range `-1 < shift < +1`?
```suggestion
<option><on>-xshift</on><od>shift the plot horizontally by a proportion of the plotting region (+/- 1.0)</od>
</option>
<option><on>-yshift</on><od>shift the plot vertically by a proportion of the plotting region (+/- 1.0)</od>
</option>
```